lib/mumukit/content_type/md_emoji.rb

```diff
@@ -20,102 +20,114 @@
+    flag-tc flag-td flag-tf flag-tg flag-th flag-tj flag-tk flag-tl flag-tm flag-tn flag-to flag-tr flag-tt flag-tv flag-tw flag-tz flag-ua
+    flag-ug flag-um flag-us flag-uy flag-uz flag-va flag-vc flag-ve flag-vg flag-vi flag-vn flag-vu flag-wf flag-ws flag-xk flag-ye flag-yt
+    flag-za flag-zm flag-zw flags flashlight fleur_de_lis flipper floppy_disk flower_playing_cards flushed fog foggy football footprints
+    fork_and_knife fountain four four_leaf_clover fr frame_with_picture free fried_shrimp fries frog frowning fu fuelpump full_moon
+    full_moon_with_face funeral_urn game_die gb gear gem gemini ghost gift gift_heart girl globe_with_meridians goat goberserk godmode
+    golf golfer grapes green_apple green_book green_heart grey_exclamation grey_question grimacing grin grinning guardsman guitar gun
+    haircut hamburger hammer hammer_and_pick hammer_and_wrench hamster hand handbag hankey hash hatched_chick hatching_chick headphones
+    hear_no_evil heart heart_decoration heart_eyes heart_eyes_cat heartbeat heartpulse hearts heavy_check_mark heavy_division_sign
+    heavy_dollar_sign heavy_exclamation_mark heavy_heart_exclamation_mark_ornament heavy_minus_sign heavy_multiplication_x heavy_plus_sign
+    helicopter helmet_with_white_cross herb hibiscus high_brightness high_heel hocho hole honey_pot honeybee horse horse_racing hospital
+    hot_pepper hotdog hotel hotsprings hourglass hourglass_flowing_sand house house_buildings house_with_garden hugging_face hurtrealbad
+    hushed ice_cream ice_hockey_stick_and_puck ice_skate icecream id ideograph_advantage imp inbox_tray incoming_envelope information_desk_person
+    information_source innocent interrobang iphone it izakaya_lantern jack_o_lantern japan japanese_castle japanese_goblin japanese_ogre
+    jeans joy joy_cat joystick jp kaaba key keyboard keycap_star keycap_ten kimono kiss kissing kissing_cat kissing_closed_eyes kissing_face
+    kissing_heart kissing_smiling_eyes knife knife_fork_plate koala koko kr label lantern large_blue_circle large_blue_diamond large_orange_diamond
+    last_quarter_moon last_quarter_moon_with_face latin_cross laughing leaves ledger left_luggage left_right_arrow left_speech_bubble
+    leftwards_arrow_with_hook lemon leo leopard level_slider libra light_rail lightning lightning_cloud link linked_paperclips lion_face
+    lips lipstick lock lock_with_ink_pen lollipop loop loud_sound loudspeaker love_hotel love_letter low_brightness lower_left_ballpoint_pen
+    lower_left_crayon lower_left_fountain_pen lower_left_paintbrush m mag mag_right mahjong mailbox mailbox_closed mailbox_with_mail
+    mailbox_with_no_mail man man-heart-man man-kiss-man man-man-boy man-man-boy-boy man-man-girl man-man-girl-boy man-man-girl-girl
+    man-woman-boy man-woman-boy-boy man-woman-girl man-woman-girl-boy man-woman-girl-girl man_in_business_suit_levitating man_with_gua_pi_mao
```

Comment:
I see some inconsistencies in naming: some emojis are `snake_case` , while others are `dash-case`. 

All them should be `snake_case`.